### ISSUE TITLE
I can't use python to create a MaskResize layer

### ISSUE THREAD

**Issue by moliqian** [BestSonny/SSTD#14]
i build caffe src code with maskresize  layer  successfully. but  i can't  use python to create a MaskResize layer.  Could you tell me what should i do to correct it ?
Following is my python code:
###             name = '{}_mask_resize'.format(from_layers[i])
            mask_resize_param = {
                    'output_height': 1,
                    'output_width': 1,
                    'factor_height': factors[i],
                    'factor_width': factors[i],
                    }
            net[name] = L.MaskRisize(net.slice1, mask_resize_param=mask_resize_param)

run the python code , it shows ：
### I0108 23:21:11.590718 20850 layer_factory.hpp:77] Creating layer conv4_3_mask_resize
F0108 23:21:11.590739 20850 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: MaskRisize (known types: AbsVal, Accuracy, AnnotatedData, AnnotatedDataMask, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, DetectionEvaluate, DetectionOutput, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MaskPooling, MaskResize, MemoryData, MultiBoxLoss, MultinomialLogisticLoss, Normalize, PReLU, Parameter, Permute, Pooling, Power, PriorBox, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, VideoData, WindowData)
*** Check failure stack trace: ***

**Comment by BestSonny** [comment]
@moliqian You created a typo MaskRisize (the correct version should be MaskResize)

**Comment by moliqian** [comment]
why are you so smart? Thank you very much!
